### PR TITLE
Add Arctic Wolf Networks blog post

### PR DESCRIPTION
**Pull request by ryandoowan**

### CODE REVIEW

File: _posts/2023-07-05-trino-fest-2023-arcticwolf.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+title: "AWS Athena (Trino) in the cybersecurity space"
+author: "Anas Shakra, Ryan Duan"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/ArcticWolf.png
+show_pagenav: false
+---
+
+AWS Athena has become a new and efficient service for companies to query their
+data using Trino. AWS Athena is a serverless, interactive analytics service
+built on open-source frameworks, supporting open-table and file formats and
```
Comment (mosabua) [inline _posts/2023-07-05-trino-fest-2023-arcticwolf.md:12]:
```suggestion
built on open-source frameworks, supporting open table and file formats and
```

File: _posts/2023-07-05-trino-fest-2023-arcticwolf.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+provides a simplified, flexible way to analyze petabytes of data where it lives.
+Senior software developer Anas Shakra from Arctic Wolf Networks gave a talk at
+[Trino Fest 2023]({% post_url 2023-06-20-trino-fest-2023-recap %}) detailing
+their switch to AWS Athena and how "queries that took hours with old solution
+now take around a minute today." Tune in to the talk or you can read the recap below!
```
Comment (mosabua) [inline _posts/2023-07-05-trino-fest-2023-arcticwolf.md:17]:
```suggestion
now take around a minute today". Tune in to the talk or you can read the recap.
```

File: _posts/2023-07-05-trino-fest-2023-arcticwolf.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+{% youtube WCuJaW7zC8k %}
+
+## Recap
+
+Generally, data access use-cases fall under three categories: investigations,
+compliance, and customer self-serve platform. The process of preparing the data
+follows an established pattern of starting with datastore, performing an
+operation to filter or transform the data, and then outputting the data in some
+form of format like a CSV or JSON depending on the client needs. However, their
```
Comment (mosabua) [inline _posts/2023-07-05-trino-fest-2023-arcticwolf.md:29]:
```suggestion
format like a CSV or JSON, depending on the client needs. However, their
```

File: _posts/2023-07-05-trino-fest-2023-arcticwolf.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+their switch to AWS Athena and how "queries that took hours with old solution
+now take around a minute today." Tune in to the talk or you can read the recap below!
+
+<!--more-->
+
+{% youtube WCuJaW7zC8k %}
+
+## Recap
+
+Generally, data access use-cases fall under three categories: investigations,
```
Comment (mosabua) [inline _posts/2023-07-05-trino-fest-2023-arcticwolf.md:25]: Add a sentence about what Arctic Wolf does before this paragraph.

Comment (mosabua) [reply _posts/2023-07-05-trino-fest-2023-arcticwolf.md:25]: Or even better add it to the first paragraph of this blog post about the talk ..

File: _posts/2023-07-05-trino-fest-2023-arcticwolf.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+form of format like a CSV or JSON depending on the client needs. However, their
+legacy service was unable to match the growing service demand and
+had four main problems:
+
+* Optimized for breadth over depth
+* Struggles to handle growing service demand
+* Proprietary query language
+* Complicated design
+
+This compelled Anas' team to find a different and improved service: AWS Athena.
```
Comment (mosabua) [inline _posts/2023-07-05-trino-fest-2023-arcticwolf.md:38]: Maybe say "Trino as provided by AWS Athena"

File: _posts/2023-07-05-trino-fest-2023-arcticwolf.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+They had four main objectives for the new service: defined access patterns,
+performant at scale, user-friendly, and deterministic pricing. AWS Athena
+satisfied these objectives, while also providing numerous benefits such as using
+a powerful query engine known as Trino, being purposefully built for large
```
Comment (mosabua) [inline _posts/2023-07-05-trino-fest-2023-arcticwolf.md:43]: reword keeping in mind that readers her already know Trino or are on the actual website and can look up more here

File: _posts/2023-07-05-trino-fest-2023-arcticwolf.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+datasets, using SQL syntax, and having a clear pricing structure. However, with
+these benefits come some drawbacks. These included subjecting to quota limits,
```
Comment (mosabua) [inline _posts/2023-07-05-trino-fest-2023-arcticwolf.md:45]: Clarify that those drawbacks are from Athena. .. not Trino.

File: _posts/2023-07-05-trino-fest-2023-arcticwolf.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+having suboptimal file sizes for their datastore, and unable to control access
```
Comment (mosabua) [inline _posts/2023-07-05-trino-fest-2023-arcticwolf.md:46]: filesize is problem of their system

File: _posts/2023-07-05-trino-fest-2023-arcticwolf.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+in their preferred defined patterns. Anas addresses this by using log queries
```
Comment (mosabua) [inline _posts/2023-07-05-trino-fest-2023-arcticwolf.md:47]: not sure what that means "control access in their preferred defined patterns" ?

File: _posts/2023-07-05-trino-fest-2023-arcticwolf.md [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+that resolves these three main impediments. These changes drastically improved their
+performance and they still have major improvements yet to resolve that might
+further enhance their performance.
```
Comment (mosabua) [inline _posts/2023-07-05-trino-fest-2023-arcticwolf.md:50]: Also maybe add that they are considering moving to Trino itself and run it on their own ..

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _posts/2023-07-03-trino-fest-2023-ibis.md [hunk 11]
```diff
@@ -8,7 +8,7 @@ show_pagenav: false
 ---
 
 The PyData stack has been described as "unreasonably effective," empowering its
-users to glean insights and analyze moderate amounts of data with a high level
```
Comment (mosabua) [inline _posts/2023-07-03-trino-fest-2023-ibis.md:11]: unrelated change - remove

File: _posts/2023-07-03-trino-fest-2023-ibis.md [hunk 12]
```diff
@@ -20,7 +20,7 @@ better: Ibis.
 
 {% youtube JMUtPl-cMRc %}
 
-A major problem with bridging the gap between Python and SQL engines has been
```
Comment (mosabua) [inline _posts/2023-07-03-trino-fest-2023-ibis.md:23]: same as above

File: _posts/2023-07-05-trino-fest-2023-arcticwolf.md [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+author: "Anas Shakra, Ryan Duan"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/ArcticWolf.png
+show_pagenav: false
+---
+
+AWS Athena has become a new and efficient service for companies to query their
+data using Trino. AWS Athena is a serverless, interactive analytics service
+built on open-source frameworks, supporting open table and file formats and
+provides a simplified, flexible way to analyze petabytes of data where it lives.
```
Comment (mosabua) [inline _posts/2023-07-05-trino-fest-2023-arcticwolf.md:13]:
```suggestion
providing a simplified, flexible way to analyze petabytes of data where it lives.
```

File: _posts/2023-07-05-trino-fest-2023-arcticwolf.md [hunk 14]
```diff
@@ -0,0 +1,64 @@
...
+
+{% youtube WCuJaW7zC8k %}
+
+## Recap
+
+Generally, data access use-cases fall under three categories: investigations,
+compliance, and customer self-serve platform. The process of preparing the data
+follows an established pattern of starting with datastore, performing an
+operation to filter or transform the data, and then outputting the data in some
+format like a CSV or JSON, depending on the client needs. However, their legacy
```
Comment (mosabua) [inline _posts/2023-07-05-trino-fest-2023-arcticwolf.md:32]:
```suggestion
format like a CSV or JSON, depending on the client needs. Arctic Wolf's custom legacy
```

File: _posts/2023-07-05-trino-fest-2023-arcticwolf.md [hunk 15]
```diff
@@ -0,0 +1,64 @@
...
+
+This compelled Anas' team to find a different and improved service: Trino as
+provided by AWS Athena.
+
+They had four main objectives for the new service: defined access patterns,
+performant at scale, user-friendly, and deterministic pricing. AWS Athena
+satisfied these objectives, while also providing numerous benefits such as using
+a powerful query engine, being purposefully built for large datasets, using SQL
+syntax, and having a clear pricing structure. However, with these benefits come
+some drawbacks for Athena. These included subjecting to quota limits, having
```
Comment (mosabua) [inline _posts/2023-07-05-trino-fest-2023-arcticwolf.md:49]:
```suggestion
some drawbacks for Athena. These includes being subject to quota limits, having
```

File: _posts/2023-07-05-trino-fest-2023-arcticwolf.md [hunk 16]
```diff
@@ -0,0 +1,64 @@
...
+suboptimal file sizes for their system, and unable to control access to internal
```
Comment (mosabua) [inline _posts/2023-07-05-trino-fest-2023-arcticwolf.md:50]:
```suggestion
suboptimal file sizes for their system, and being unable to control access sufficiently.
```

File: _posts/2023-07-05-trino-fest-2023-arcticwolf.md [hunk 17]
```diff
@@ -0,0 +1,64 @@
...
+and external clients. Anas addresses this by using log queries that resolves
```
Comment (mosabua) [inline _posts/2023-07-05-trino-fest-2023-arcticwolf.md:51]:
```suggestion
nas addresses this by using log queries that resolves
```

File: _posts/2023-07-05-trino-fest-2023-arcticwolf.md [hunk 18]
```diff
@@ -0,0 +1,64 @@
...
+these three main impediments. These changes drastically improved their further
```
Comment (mosabua) [inline _posts/2023-07-05-trino-fest-2023-arcticwolf.md:52]: Get rid of the sentence 

These changes drastically improved their further...

File: _posts/2023-07-05-trino-fest-2023-arcticwolf.md [hunk 19]
```diff
@@ -0,0 +1,64 @@
...
+enhance their performance. In addition, they are considering switching
```
Comment (mosabua) [inline _posts/2023-07-05-trino-fest-2023-arcticwolf.md:53]:
```suggestion
As next step Anas is considering switching to a self-managed Trino deployment for more control with the same performance gains.
```

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _posts/2023-07-05-trino-fest-2023-arcticwolf.md [hunk 20]
```diff
@@ -0,0 +1,62 @@
...
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/ArcticWolf.png
+show_pagenav: false
+---
+
+AWS Athena has become a new and efficient service for companies to query their
+data using Trino. AWS Athena is a serverless, interactive analytics service
+built on open-source frameworks, supporting open table and file formats and
+providing a simplified, flexible way to analyze petabytes of data where it lives.
+Arctic Wolf Networks, a cybersecurity company that provides security monitoring
```
Comment (colebow) [inline _posts/2023-07-05-trino-fest-2023-arcticwolf.md:14]: I would re-order this paragraph. Start by introducing who Arctic Wolf is (it's their talk), then explain what Athena is after mentioning that htey use it. You should also be more explicit about the fact that Athena runs on Trino.

File: _posts/2023-07-05-trino-fest-2023-arcticwolf.md [hunk 21]
```diff
@@ -0,0 +1,62 @@
...
+solution now take around a minute today". Tune in to the talk or you can read
+the recap!
+
+<!--more-->
+
+{% youtube WCuJaW7zC8k %}
+
+## Recap
+
+Generally, data access use-cases fall under three categories: investigations,
```
Comment (colebow) [inline _posts/2023-07-05-trino-fest-2023-arcticwolf.md:28]:
```suggestion
At Arctic Wolf, data access use-cases fall under three categories: investigations,
```
This isn't general, this is specific to what they do. 😛

**Review by colebow — COMMENTED** [review]
Looks good on the whole, just a couple comments.

File: _posts/2023-07-05-trino-fest-2023-arcticwolf.md (hunk 21 above)
Comment (mosabua) [reply _posts/2023-07-05-trino-fest-2023-arcticwolf.md:28]: ;-)

**Review by mosabua — APPROVED** [review]
Looks good now.

**Review by dain — APPROVED** [review]
